src/transformers/models/swiftformer/configuration_swiftformer.py

```diff
@@ -43,6 +43,8 @@ class SwiftFormerConfig(PretrainedConfig):
 
 
     Args:
+        image_size (`int`, *optional*, defaults to 224):
+            The size (resolution) of each image
```

Comment:
What was the reason for adding these parameters to the config when they weren't present before?

Reply:
I think most were added because of these comments: https://github.com/huggingface/transformers/pull/23342#discussion_r1453860652 and https://github.com/huggingface/transformers/pull/23342#discussion_r1453861951. Regarding the `image_size` parameter I don't remember clearly, but I think it was because I believed that the usual convention was to have `image_size` as a parameter on most models. Is this line of thought wrong?

Reply:
No, this is okay - I'll defer to @amyeroberts on that one!